src/app/ecosystem-explorer/[slug]/components/VideoSection.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+
+import LiteYouTubeEmbed from 'react-lite-youtube-embed'
+import useSWR from 'swr'
+
+import { fetchYouTubeVideoMetadata } from '../services/youtube'
+
+const YOUTUBE_EMBED_BASE_URL = 'https://www.youtube.com/embed/'
+
+type YouTubeEmbedProps = {
+  videoUrl: string
+}
+
+const removeMoreVideosSuggestionsOnPause = 'rel=0'
+
+export function VideoSection({ videoUrl }: YouTubeEmbedProps) {
+  const isUrlFormatValid = videoUrl.startsWith(YOUTUBE_EMBED_BASE_URL)
+  const videoId = videoUrl.replace(YOUTUBE_EMBED_BASE_URL, '')
+
+  const { data, error } = useSWR(
+    isUrlFormatValid ? videoId : null,
+    fetchYouTubeVideoMetadata,
+    {
+      revalidateOnFocus: false,
+      revalidateOnReconnect: false,
+      shouldRetryOnError: false,
+    },
+  )
+
+  if (!isUrlFormatValid || error) {
+    return (
+      <div
+        className="flex aspect-video items-center justify-center rounded-lg bg-brand-700"
+        role="status"
+      >
+        <p>Video unavailable</p>
+      </div>
+    )
+  }
```

Comment:
Did we ask @filipagr to look into the design for this? 🙏🏼

Reply:
Not yet, but I'm planning on it :)

Reply:
https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1721810299369289